### PR TITLE
Switch app name and page name in title (bsc#967423)

### PR DESCRIPTION
**Pull request by rsalevsky**
title

### CODE REVIEW

**Comment by vuntz** [comment]
@kwwii is this fine with you?

File: crowbar_framework/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -38,10 +38,14 @@ def branding_config
 
   # Generate the meta title that gets displayed on the page meta information
   def meta_title
-    title = [
-      branding_config[:page_title],
-      branding_config[:page_slogan]
-    ].compact.join(" ")
+    if branding_config[:page_slogan].blank?
```
Comment (tboerger) [inline crowbar_framework/app/helpers/application_helper.rb]: Please keep it like it was.

**Comment by tboerger** [comment]
One change I would reject, switching the title is totally fine.

**Comment by kwwii** [comment]
@rsalevsky : yes, this looks fine to me.

**Comment by MaximilianMeister** [comment]
+1

**Comment by tboerger** [comment]
:+1:

**Comment by tboerger** [comment]
I think you have to change the page titles for the error pages as well.

**Comment by vuntz** [comment]
+1.

@tboerger are you +1 after the error pages got changed?

**Comment by tboerger** [comment]
:+1:

**Pull request merged**